lib/rules/property-no-unknown/__tests__/index.mjs

```diff
@@ -378,23 +386,11 @@ testRule({
 			line: 1,
 			column: 21,
 		},
-		{
-			code: '@not-my-at-rule { foo: 1; }',
-			message: messages.rejected('foo'),
-			line: 1,
-			column: 19,
-		},
```

Comment:
We can't tell whether declarations within custom at-rules are property or descriptor ones, so we now ignore these potential false positives.